### PR TITLE
Honor user page size preference in new system list pages

### PR DESCRIPTION
**Pull request by cbosdo**
## What does this PR change?

Take user preferences into account in page size of (virtual) systems lists.

## GUI diff

No difference.

- [X] **DONE**

## Documentation
- No documentation needed: only internal and user invisible changes
- [X] **DONE**

## Test coverage
- No tests: Do we really want integration tests for such a tiny change?

- [X] **DONE**

## Links

Fixes https://github.com/uyuni-project/uyuni/issues/6188

- [X] **DONE**

## Changelogs

Make sure the changelogs entries you are adding are compliant with https://github.com/uyuni-project/uyuni/wiki/Contributing#changelogs and https://github.com/uyuni-project/uyuni/wiki/Contributing#uyuni-projectuyuni-repository

If you don't need a changelog check, please mark this checkbox:

- [ ] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_pgsql_tests"
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"
- [ ] Re-run test "spacecmd_unittests"

### CODE REVIEW

File: web/html/src/manager/systems/all-list.tsx [hunk 1]
```diff
@@ -68,6 +69,7 @@ export function AllSystems(props: Props) {
         searchField={<SearchField options={allListOptions} name="criteria" />}
         defaultSearchField={props.queryColumn || "server_name"}
         initialSearch={props.query}
+        initialItemsPerPage={props.pageSize}
```
Comment (Etheryte) [inline web/html/src/manager/systems/all-list.tsx:72]: Here and elsewhere, setting this prop on a `Table` should not be needed, `TableDataHandler` already uses `window.userPrefPageSize` as the default if it's set.

Comment (cbosdo) [reply web/html/src/manager/systems/all-list.tsx:72]: Oh OK! IIRC Luis asked me a long time ago to start getting away from  that global `window` variable... does it mean I can just forget about that now?

Comment (Etheryte) [reply web/html/src/manager/systems/all-list.tsx:72]: Not really, we still need to pass the value from the server to the frontend somehow. Wherever it's available on `window` it's used automatically by `TableDataHandler` though so you don't need to manually pass it around after it's set.

Comment (cbosdo) [reply web/html/src/manager/systems/all-list.tsx:72]: OK that means I probably only need the Java part of this PR

Comment (cbosdo) [reply web/html/src/manager/systems/all-list.tsx]: done

**Pull request merged**